insalan/tournament/models/seating.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class Seating(models.Model):
+    """
+    Represents a single seat
+    """
+
+    event = models.ForeignKey(
+        "Event",
+        null=False,
+        blank=False,
+        on_delete=models.CASCADE,
+        verbose_name=_("Évènement"),
+    )
+    x = models.IntegerField(
+        null=False,
+        blank=False,
+        verbose_name=_("Position horizontale"),
+    )
+    y = models.IntegerField(
+        null=False,
+        blank=False,
+        verbose_name=_("Position verticale"),
+    )
+
+    tournament = models.ForeignKey(
+        "Tournament",
+        null=True,
+        blank=True,
+        on_delete=models.CASCADE,
+        verbose_name=_("Tournoi"),
+    )
+    team = models.ForeignKey(
+        "Team",
+        null=True,
+        blank=True,
+        on_delete=models.CASCADE,
```

Comment:
:warning: You can't use on_delete with cascade here. If a team owning a seat is deleted, that will delete the seat too. You need to use models.SET_NULL to remove the team.